unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/Catalog/QuickBarController.cs

```diff
@@ -0,0 +1,36 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class QuickBarController 
+{
+    public event System.Action<SceneObject> OnSceneObjectSelected;
+
+    SceneObject[] quickBarShortcutsSceneObjects = new SceneObject[9];
```

Comment:
I would have this in a const. It would also allow us to have a really tiny integration test that compares that the view and the controller are in sync with the same amount of slots.